ecs-init/config/common.go

```diff
@@ -247,6 +247,11 @@ func CgroupMountpoint() string {
 	return cgroupMountpoint
 }
 
+// MountDirectoryEBS returns the location on disk where EBS volumes will be mounted
+func MountDirectoryEBS() string {
+       return directoryPrefix + "/mnt/ecs/ebs"
```

Comment:
Q: is `directoryPrefix` set somewhere? Perhaps we should add a comment of what a full directory path here for reference.

Reply:
yes it's set in release.go
/ecs-init/config/release.go:   `directoryPrefix = ""`

(this is only set if we are building with an alternate base directory)